README.md

```diff
@@ -16,10 +16,31 @@ To build, install Stata locally, apply your licence, and then copy the
 following into the `bin/` subdirectory:
 
 * `/usr/local/stata/bin/stata*`
+* `/usr/local/stata/bin/stinit`
 * `/usr/local/stata/bin/ado/`
 * `/usr/local/stata/utilities/`
 
+
 ## Relicensing Stata
 
-To relicense, apply the licence locally, and then copy
-`/usr/local/stata/stata.lic` into `bin/`.
+The docker image contains the machinery needed to update the license. To
+generate a new license file, first obtain the new serial number, code, and
+authorisation. Then run:
+
+    docker run --rm -v $PWD:/src -w /usr/local/stata --entrypoint /src/scripts/renew-license.sh ghcr.io/opensafely-core/stata-mp "SERIAL" "CODE" "AUTH"
+
+The resulting file will be copied to `./stata.lic` (note: due to docker
+shenanigans, it will be owned by root). You can test this works with:
+
+    ./scripts/test-license.sh
+
+The contents of this license file can then be used to update the `STATA_LICENSE`
+env var for job-runner. You should also update the `stata.lic` file in
+[https://github.com/opensafely/server-instructions](https://github.com/opensafely/server-instruction)
```

Comment:
I couldn't remember what `server-instructions` is -- could you add a short README in that repo?

Reply:
Hmm, this is a bit tricky.

I believe originally it served as docs for how to access TPP server, but it's badly out of date.

I stuck the license file in there as I needed a private repo that only opensafely org members could access, so they can grab our stata license locally when needed.

So I'm not 100% sure what the read me should say, other than ^

Reply:
I think that's better than nothing :D

Reply:
done